README.md

```diff
@@ -41,6 +41,11 @@ Developers are expected to run the app locally using [docker-compose](https://do
 * `make build` / `docker compose build` will rebuild the Funding Service image.
 * `make clean-build` / `docker compose build --no-cache` will rebuild the Funding Service image, bypassing caching. This should rarely be needed.
 
+### SSO
+By default, local development and locally-run end to end tests use a [stub server](./stubs/sso/) to implement the same  interface as Microsoft SSO flow. This is so we can reliably run e2e tests locally without hitting the real SSO endpoints.
+
+If you need to allow real SSO login locally, you can set an environment variable of `AZURE_AD_LOGIN_DOMAIN=https://login.microsoftonline.com/` in your .env file, or comment out the LocalConfig version of this variable in the config so it falls back to the BaseConfig default. Be aware that the SSO e2e test will fail while this is enabled - don't forget to switch back to check this passes.
```

Comment:
If they need real SSO locally, they also need to get the 3 Azure AD creds from our 1password.

Reply:
When we update the e2e test to work with real MHCLG SSO we should remember to come back and remove this caveat.